src/managers/GoogleCalendarManager.ts

```diff
@@ -88,9 +88,12 @@ export default class {
             // We only send embeds for events that are just starting in our time window.
             if (searchInterval.contains(startTime)) {
               const mentions = this.meetingPingsSchema.getMentions(calendarID, event);
+              // Replace all br tags with newlines and remove all other HTML tags.
+              const description = event.description?.replace(/<br>/g, '\n')?.replace(/<[^<]*>|&[^&]*;/g, '');
```

Comment:
Additionally, it'd be helpful if the comment here describes in more detail what the tag regex is doing so it's easier to read/maintain in the future.